In [1]:
import pandas as pd

df = pd.read_csv('Dataset/international_matches.csv')

print(df.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 23921 entries, 0 to 23920
Data columns (total 25 columns):
 #   Column                         Non-Null Count  Dtype  
---  ------                         --------------  -----  
 0   date                           23921 non-null  object 
 1   home_team                      23921 non-null  object 
 2   away_team                      23921 non-null  object 
 3   home_team_continent            23921 non-null  object 
 4   away_team_continent            23921 non-null  object 
 5   home_team_fifa_rank            23921 non-null  int64  
 6   away_team_fifa_rank            23921 non-null  int64  
 7   home_team_total_fifa_points    23921 non-null  int64  
 8   away_team_total_fifa_points    23921 non-null  int64  
 9   home_team_score                23921 non-null  int64  
 10  away_team_score                23921 non-null  int64  
 11  tournament                     23921 non-null  object 
 12  city                           23921 non-null 

In [2]:
import pandas as pd

df = pd.read_csv('Dataset/international_matches.csv')
num_row=len(df)

print(num_row)

23921


#Goal of Collecting Dataset:

The primary goal of collecting and analyzing this dataset is to gain insights into the FIFA World Cup 2022 matches. This can involve tasks such as:

Performance Analysis: Evaluating team and player performance based on various metrics.
Predictive Modeling: Developing models to predict match outcomes or player performance.
Trend Analysis: Identifying trends and patterns in match results, scoring, and other relevant statistics.

We will achieve this by clustering data and classifying it using machine learning techniques acquired during the course.


##Source of Dataset:
Kaggle Platform Dataset link: https://www.kaggle.com/datasets/brenda89/fifa-world-cup-2022

##General Information about the Dataset:

Number of Attributes:25

Number of Objects:23921 (Match)

Class Name/Labels: home_team_result
